test/test_simulation_results.jl

```diff
@@ -139,7 +147,7 @@ function test_simulation_results(file_path::String)
             count = 25,
         )
 
-        clear_simulation_results!(results_ed)
+        empty!(results_ed)
```

Comment:
is it preferable to use Base.empty! ?

Reply:
I don't think there is a correct answer. The Julia standard library uses `empty` rather than `clear` for this sort of thing, so it should probably be `empty_simulation_results`, `empty_results`, or just `empty`. We probably are not consistent on this point.

We are storing containers of results, and so `empty!(results)` felt natural to me. I will go with whatever you prefer.

Reply:
Base.empty! is ok to me. I didn't realize this was the change when I saw the other change.